src-11ty/_includes/base.html

```diff
@@ -0,0 +1,192 @@
+    <style>
+      body {
+        margin: 0;
+      }
+
+      .app-header {
+        display: block;
+        background: var(--app-bg-header);
+        position: sticky;
+        top: 0;
+        z-index: 100;
+      }
+
+      .app-header header {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        padding: 0 20px;
+        border-bottom: 1px solid var(--app-border);
+        height: 64px;
+        margin: 0 auto;
+      }
+
+      .app-header .logo-section {
+        display: flex;
+        align-items: center;
+        gap: 12px;
+        cursor: pointer;
+      }
+
+      .app-header .logo {
+        height: 32px;
+        width: auto;
+      }
+
+      .app-header .title {
+        font-size: 1.25rem;
+        font-weight: 500;
+        color: var(--app-text-primary);
+        margin: 0;
+      }
+
+      .app-header .nav-links {
+        display: flex;
+        gap: 20px;
+        align-items: center;
+      }
+
+      .app-header .nav-link {
+        color: var(--app-text-secondary);
+        border: 1px solid var(--app-border);
+        text-decoration: none;
+        padding: 8px 12px;
+        border-radius: 6px;
+        transition: all 0.2s ease;
+        font-weight: 400;
+      }
+
+      .app-header .nav-link:hover {
+        background: var(--app-bg-secondary);
+        color: var(--app-text-primary);
+        border-color: var(--app-border-hover);
+      }
+
+      .app-header .github-link {
+        display: flex;
+        align-items: center;
+        gap: 6px;
+        color: #666;
+        text-decoration: none;
+        padding: 8px 12px;
+        border-radius: 6px;
+        transition: all 0.2s ease;
+        border: 1px solid #e0e0e0;
+      }
+
+      .app-header .github-link:hover {
+        background: #f5f5f5;
+        color: #333;
+        border-color: #ccc;
+      }
+
+      .app-header .github-icon {
+        width: 16px;
+        height: 16px;
+        fill: currentColor;
+      }
+
+      @media (max-width: 768px) {
+        .app-header .nav-links {
+          gap: 12px;
+        }
+
+        .app-header .nav-link,
+        .app-header .github-link {
+          padding: 6px 8px;
+          font-size: 0.9rem;
+        }
+
+        .app-header .title {
+          font-size: 1.1rem;
+        }
+
+        .app-header .github-link .github-text {
+          display: none;
+        }
+      }
+
+      @media (max-width: 480px) {
+        .app-header header {
+          padding: 0 16px;
+        }
+
+        .app-header .logo {
+          height: 28px;
+        }
+      }
+
+      details-page:not(:defined) {
+        opacity: 0;
+      }
+
+      details-page {
+        opacity: 1;
+        transition: opacity 0.1s;
+      }
+    </style>
+  </head>
+  <body class="wa-palette-shoelace wa-theme-shoelace">
+    <div class="app-header">
+      <header>
+        <div class="logo-section" onclick="window.location.href = '/'">
```

Comment:
This should be an `a` tag. JavaScript navigation is not accessible, and it cannot be opened in a new tab etc.